exts/cesium.usd.plugins/schemas/cesium_schemas.usda

```diff
@@ -11,9 +11,10 @@
 
 over "GLOBAL" (
     customData = {
-        string libraryName       = "cesium"
+        string libraryName       = "CesiumUsdSchemas"
```

Comment:
I was debating between `cesium`, `cesiumUsdSchemas`, and `CesiumUsdSchemas`. From a technical perspective it doesn't seem to matter (the underscore is just a convention) so I went with uppercase since it's the most consistent everywhere.

**Before**

libraryName|plugInfo.json|moduleDeps|include path|.pyd|.dll|defines|python module
--|--|--|--|--|--|--|--
`cesium`|`CesiumUsdSchemas`|`cesiumUsdSchemas`|`CesiumUsdSchemas/tileset.h`|`_cesiumUsdSchemas.pyd`|`CesiumUsdSchemas.dll`|`CESIUM_API`|`CesiumUsdSchemas`

**After**

libraryName|plugInfo.json|moduleDeps|include path|.pyd|.dll|defines|python module
--|--|--|--|--|--|--|--
`CesiumUsdSchemas`|`CesiumUsdSchemas`|`CesiumUsdSchemas`|`CesiumUsdSchemas/tileset.h`|`_CesiumUsdSchemas.pyd`|`CesiumUsdSchemas.dll`|`CESIUMUSDSCHEMAS_API`|`CesiumUsdSchemas`